spec.md

```diff
@@ -270,7 +270,19 @@ An asynchronous response triggers the platform marketplace to poll the endpoint
 
 #### Blocking Operations
 
-The marketplace MUST ensure that service brokers do not receive requests for an instance while an asynchronous operation is in progress. For example, if a broker is in the process of provisioning an instance asynchronously, the marketplace can not allow any update, bind, unbind, or deprovision requests to be made through the platform. A user who attempts to perform one of these actions while an operation is already in progress MUST receive a `400 Bad Request` response with the error message: `Another operation for this service instance is in progress`.
+This specification places no requirement on brokers with respect to their
+support for processing concurrent requests that act upon the same set of
+resources. If a broker receives a request that it is not able to process
+due to other activity being done then the broker MUST reject the request
+with an HTTP `400 Bad Request` response. The HTTP body of the response MUST
```

Comment:
The convention elsewhere in the spec is to return a 422 if a broker can't perform something that some brokers can. Unless we say that broker MUST not support parallel operations, then a 400 doesn't seem appropriate.